include/boost/math/tools/polynomial.hpp

```diff
@@ -338,13 +327,12 @@ class polynomial
       }
    }
 #ifdef BOOST_MATH_HAS_IS_CONST_ITERABLE
```

Comment:
I suspect BOOST_MATH_HAS_IS_CONST_ITERABLE is always set in C++11 land?

Other than that, this all looks good to me,, thanks!

Reply:
BOOST_MATH_HAS_IS_CONST_ITERABLE requires cxx14_variable_templates so it has to stay put for now.

Reply:
Ah, shame.  No matter.